examples/utils.js

```diff
@@ -0,0 +1,68 @@
+import { check, fail } from "k6";
+import { randomSeed } from "k6";
+import http from "k6/http";
+import vars from "./vars.js";
+
+let authToken = __ENV.TOKEN;
+let seed = Number(__ENV.RANDOM_SEED);
+
+export function getRandom(max) {
+  return Math.floor(Math.random() * max) + 1;
+}
+
+export function setRandomSeed() {
+  seed = seed || getRandom(Number.MAX_SAFE_INTEGER);
+  randomSeed(seed);
+  console.info(`Generated random seed: ${seed}`);
+}
+
+export function respCheck(resp, opts) {
+  const { httpCode = 200, bastidaCode = 200, dataChecker = null, needFail = true } = opts || {};
+  let checks = {
+    "is status ok": (r) => r.status === httpCode,
+  };
+  if (bastidaCode) {
+    checks["is result code ok"] = (r) => r.body && r.json().status.code === bastidaCode;
+  }
+  if (dataChecker) {
+    checks["are returned data expected"] = (r) => r.body && dataChecker(r.json().data);
+  }
+  let result = check(resp, checks);
+  if (!result) {
+    console.warn(resp.body);
+    if (needFail) fail();
+  }
+  return result;
+}
+
+export function getBaseUrl() {
+  let node = typeof __ITER == "undefined" ? vars.hosts[0] : vars.hosts[__ITER % vars.hosts.length];
+  return `http://${node}`;
+}
+
+function getAuthToken(expired_at) {
```

Comment:
Function arguments are `camelCase`. And it is unused btw.
```suggestion
function getAuthToken(expiresAt) {
```
Simple [Google's TS naming conventions](https://google.github.io/styleguide/tsguide.html#naming-rules-by-identifier-type) may be helpful